### PR TITLE
Fix stylelint

### PR DESCRIPTION
**Pull request by neolitec**
Signed-off-by: Kevin Manson kevin.manson@corp.ovh.com

### CODE REVIEW

File: packages/oui-checkbox/_mixins.less [hunk 1]
```diff
@@ -221,7 +221,7 @@
     }
 
     .@{checkbox-selector}__input:not(:disabled) + .@{checkbox-selector}__label-container:hover,
-    .@{checkbox-selector}__input:checked:not(:disabled) + .@{checkbox-selector}__label-container:hover {
+      .@{checkbox-selector}__input:checked:not(:disabled) + .@{checkbox-selector}__label-container:hover {
```
Comment (jimmyfortinx) [inline packages/oui-checkbox/_mixins.less]: Is this really something stylelint expect ? Not sure to like this additionnal indentation, I know it's purely personal...

File: packages/oui/_settings.less [hunk 2]
```diff
@@ -1,8 +1,14 @@
 // Settings - Global
 @oui-global-base-z-index: 0;
+
```
Comment (jimmyfortinx) [inline packages/oui/_settings.less]: why adding empty lines between each variables ?

File: packages/oui-checkbox/_mixins.less (hunk 1 above)
Comment (neolitec) [inline packages/oui-checkbox/_mixins.less]: Yes, stylelint expects this style of indentation...

File: packages/oui/_settings.less (hunk 2 above)
Comment (neolitec) [inline packages/oui/_settings.less]: Oh, I used stylefmt. Please do not use it!

File: packages/oui-checkbox/_mixins.less (hunk 1 above)
Comment (jimmyfortinx) [inline packages/oui-checkbox/_mixins.less]: Saw together, it's a problem of `postcss-less` and `stylelint`. You can get more information here: https://github.com/stylelint/stylelint/issues/1741

File: packages/oui-list/_mixins.less [hunk 3]
```diff
@@ -197,7 +197,7 @@
       border-bottom: @separator-size solid @separator-color;
 
       &:last-child {
-          border-bottom: 0 none;
+        border-bottom: 0 none;
```
Comment (NicoPennec) [inline packages/oui-list/_mixins.less]: none is enough: `border-bottom: 0 none` => `border-bottom: none`

File: packages/oui-progress/_mixins.less [hunk 4]
```diff
@@ -104,11 +104,11 @@
...
 
   @keyframes animate-stripes {
-      from {
-        background-position-x: 0px;
-      }
-      to {
-        background-position-x: @oui-progress-height * 2;
-      }
+    from {
+      background-position-x: 0px;
```
Comment (NicoPennec) [inline packages/oui-progress/_mixins.less]: useless unit: `0px` => `0`

File: packages/oui-radio/_mixins.less [hunk 5]
```diff
@@ -328,7 +328,7 @@
       }
 
       ~ .@{selector}__label-container::before,
-        &:checked ~ .@{selector}__label-container::before {
+      &:checked ~ .@{selector}__label-container::before {
         box-shadow: 0px 0px 3px 1px rgba(102, 199, 214, 0.6);
```
Comment (NicoPennec) [inline packages/oui-radio/_mixins.less]: ueless unit: `0px` => `0`

File: packages/oui-radio/_mixins.less [hunk 6]
```diff
@@ -423,8 +423,8 @@
         }
 
         .@{selector}__input:not(:disabled):focus + .@{selector}__label-container,
-        .@{selector}__input:checked:not(:disabled):focus + .@{selector}__label-container {
-            box-shadow: 0px 0px 3px 0px @@status-border-color;
+          .@{selector}__input:checked:not(:disabled):focus + .@{selector}__label-container {
+          box-shadow: 0px 0px 3px 0px @@status-border-color;
```
Comment (NicoPennec) [inline packages/oui-radio/_mixins.less]: useless unit: `0px` => `0`

File: packages/oui-list/_mixins.less (hunk 3 above)
Comment (neolitec) [inline packages/oui-list/_mixins.less]: It's not the point here @NicoPennec but yes, `none` would be better. Maybe stylelint will be able to help us with that. Soon.

Comment (neolitec) [inline packages/oui-list/_mixins.less]: See https://github.com/ovh-ux/ovh-ui-kit/pull/39

File: packages/oui-progress/_mixins.less (hunk 4 above)
Comment (neolitec) [inline packages/oui-progress/_mixins.less]: https://github.com/ovh-ux/ovh-ui-kit/pull/39

File: packages/oui-list/_mixins.less (hunk 3 above)
Comment (neolitec) [inline packages/oui-list/_mixins.less]: Can be done by stylelint with this plugin: https://github.com/timothyneiljohnson/stylelint-value-border-zero

Comment (antleblanc) [inline packages/oui-list/_mixins.less]: :tada: Conjuring with this: https://github.com/ovh-ux/stylelint-config-ovh, it will be awesome :unicorn: !

**Pull request merged**